### DISCUSSION TITLE
Extension not working

### DISCUSSION THREAD

**Discussion by j-fedotenkovs** [lllyasviel/stable-diffusion-webui-forge#298] [Q&A] [answered]
Hi! 

First of all big up to @lllyasviel and everyone contributing for developing Forge UI. I'm having massive increase in generation speed on my 6Gb 1660Ti. 
I'd like to utilize that performance increase using Deforum for my video projects, but unfortunately when I install Deforum extension via the Extensions tab and reload the UI, Deforum tab is not showing up. I guess Deforum extension was initially developed for A1111 and there's is some compatibility issue. Maybe there is a way to modify some of the files to make Deforum tab available in Forge UI? I would appreciate any help in that direction. 

Also @lllyasviel are you planning to work on making A1111 extensions compatible with Forge AI if that's possible? Or extensions themselves need to be modified to be compatible with Forge UI?

**Comment by catboxanon** [comment, marked as answer]
Deforum is not supported currently, and is being looked into soon. https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/96#issuecomment-1950292908

**Reply by Bilt3d** [reply]
any progress on this?

**Reply by DarkNeo88** [reply]
omg just use search option https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/1754#discussioncomment-10587068
